docs/notebooks/utils/downloading_brains.ipynb

```diff
@@ -52,8 +52,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "dir = \"s3://mouse-light-viz/precomputed_volumes/brain1\"\n",
-    "dir_segments = \"s3://mouse-light-viz/precomputed_volumes/brain1_segments\"\n",
+    "dir = \"s3://open-neurodata/brainlit/brain1\"\n",
```

Comment:
Is the earlier cell, that says "We have prepared a 2 low-resolution brain volumes, as well as a single axon segment" still true?